engines/dfc_provider/app/controllers/dfc_provider/api/people_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+# Controller used to provide the People API for the DFC application
+module DfcProvider
+  module Api
+    class PeopleController < BaseController
```

Comment:
you can use persons, it's correct english and will be more obvious that it is the controller of the person object :-)

Reply:
Oh ok, didn't know about that.